test-cases/PR-provision-test-docker.yaml

```diff
@@ -14,8 +14,7 @@ nemesis_filter_seeds: false
 user_prefix: 'PR-provision-docker'
 
 scylla_version: 4.4.1
-scylla_mgmt_repo: 'http://downloads.scylladb.com/rpm/centos/scylladb-manager-2.3.repo'
-scylla_mgmt_agent_repo: 'http://downloads.scylladb.com/rpm/centos/scylladb-manager-2.3.repo'
+manager_branch: '2.3'
```

Comment:
why 2.3 and not 2.4? or either 2.5?

Reply:
Well, 2.5 is not our yet, but yeah I guess this could be promoted. @dkropachev Any objections?

Reply:
remove from here completely so it will use defaults.